swagger.yaml

```diff
@@ -0,0 +1,94 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Tech Holding Classes
+  description: An app to manage Tutorials
+host: localhost:8000
+basePath: "/"
+tags:
+- name: tutorials
+  description: API for user in the system
+schemes:
+- http
+consumes:
+- application/yaml
```

Comment:
Should be `application/json`

Reply:
This is Request and response `Content-Type` which is in our case should be always json.